tests/unit/challenges/test_views.py

```diff
@@ -443,17 +462,29 @@ def test_particular_challenge_for_disable_does_not_exist(self):
         self.url = reverse_lazy('challenges:disable_challenge',
                                 kwargs={'pk': self.challenge.pk + 2})
         response = self.client.post(self.url, {})
-        self.assertEqual(response.status_code, status.HTTP_406_NOT_ACCEPTABLE)
+        self.assertEqual(response.status_code, status.HTTP_403_FORBIDDEN)
 
     def test_when_user_does_not_have_permission_to_disable_particular_challenge(self):
         self.url = reverse_lazy('challenges:disable_challenge',
                                 kwargs={'pk': self.challenge2.pk})
         expected = {
-            'error': 'Sorry, you do not have permission to disable this challenge'
+            'detail': 'Sorry, you are not allowed to perform this operation!'
```

Comment:
The `detail` key is from the framework and alas I have no control over it. I can change the test case to only check the value instead of the whole dictionary. Would that be a good solution?